src/main/java/games/strategy/engine/framework/map/download/DownloadCoordinator.java

```diff
@@ -1,107 +1,147 @@
 package games.strategy.engine.framework.map.download;
 
-import java.util.HashMap;
+import java.util.Collection;
 import java.util.HashSet;
-import java.util.Map;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
 import java.util.Set;
-import java.util.function.Consumer;
-import java.util.function.Predicate;
+import java.util.concurrent.CopyOnWriteArrayList;
 
-import games.strategy.debug.ClientLogger;
 import games.strategy.engine.ClientFileSystemHelper;
-import games.strategy.util.ThreadUtil;
 
 /**
  * Class that accepts and queues download requests. Download requests are started in background
  * thread, this class ensures N are in progress until all are done.
  */
-public class DownloadCoordinator {
+final class DownloadCoordinator {
+  static final DownloadCoordinator INSTANCE = new DownloadCoordinator();
```

Comment:
Derp...  If `DownloadMapsWindow` owns `DownloadCoordinator`, then that defeats the purpose of this PR.  So scratch that one...  :smile: